mrp_subcontracting/models/stock_picking.py

```diff
@@ -72,8 +72,10 @@ def action_done(self):
                         })
                 else:
                     for move_line in move.move_line_ids:
+                        active_id = production and production.id or False
```

Comment:
You can simplify this:

```suggestion
                        active_id = production.id
```